src/ServiceControl.Audit/Infrastructure/SetupBootstrapper.cs

```diff
@@ -40,6 +45,35 @@ public async Task Run(string username)
 
             //No need to start the raw endpoint to create queues
             await RawEndpoint.Create(config).ConfigureAwait(false);
+
+            var configuration = new EndpointConfiguration(settings.ServiceName);
```

Comment:
This feels wrong, basically doing the RawEndpoint create and the normal Create is also run....

Reply:
Also, instead of doing this can't we then just not only create the EndpointConfiguration object and invoke `INeedInitialization` without doing `NServiceBusFactory.Create`?